### PR TITLE
domu: Introduce image extras for LITMUS-RT guest domain

### PR DESCRIPTION
**Pull request by aanisov**
Signed-off-by: Andrii Anisov <andrii_anisov@epam.com>

### CODE REVIEW

File: recipes-domu/domu-image-litmusrt/domu-image-litmusrt.bb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+SUMMARY = "ARMV8 Generic LITMUS-RT guest domain built with yocto"
```
Comment (andr2000) [inline recipes-domu/domu-image-litmusrt/domu-image-litmusrt.bb]: is there any other option Yocto recipe could be built with? ;)

**Review by lorc — APPROVED** [review]
`Reviewed-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

File: recipes-domu/domu-image-litmusrt/domu-image-litmusrt.bb (hunk 1 above)
Comment (aanisov) [reply recipes-domu/domu-image-litmusrt/domu-image-litmusrt.bb]: Yep, the comment is a bit redundantю

**Comment by aanisov** [comment]
updated

**Review by andr2000 — APPROVED** [review]

**Comment by aanisov** [comment]
@iartemenko be so kind to merge this

**Pull request merged**